src/main/java/seedu/duke/storage/TaskList.java

```diff
@@ -0,0 +1,103 @@
+package seedu.duke.storage;
+
+import java.util.ArrayList;
+
+import seedu.duke.commands.Ui;
+import seedu.duke.exceptions.action.DukeActionOutOfBoundException;
+import seedu.duke.tasks.Task;
+
+public class TaskList {
+    private final ArrayList<Task> taskList;
+
+    /**
+     * Primary Constructor.
+     */
+    public TaskList() {
+        this.taskList = new ArrayList<>();
+    }
+
+    /**
+     * Retrieves the list of tasks that is being stored in this class.
+     * 
+     * @return an ArrayList of tasks which is being stored.
+     * @see java.util.ArrayList
+     */
+    public ArrayList<Task> getTaskList() {
+        return this.taskList;
+    }
+
+    /**
+     * Loads previous saved Tasks from a txt file. Save it in this current TaskList
+     * in an ArrayList of Task object.
+     * 
+     * @param savedTasks this is the ArrayList of Task coming from the txt file.
+     * @see java.util.ArrayList
+     */
+    public void loadFromStorage(ArrayList<Task> savedTasks) {
+        this.taskList.addAll(savedTasks);
+    }
+
+    /**
+     * Helps to mark the task in the list of Tasks as done.
```

Comment:
Is there a reason why some of your JavaDocs comments use `Helps..`? Would it be easier/clearer to just say `Marks...` etc.?